Library/src/Systems/Log/LogSystem.cpp

```diff
@@ -150,4 +150,9 @@ void LogSystem::ClearAllCallbacks()
 	Callbacks->Clear();
 }
 
+void LogSystem::LogTestMessage()
```

Comment:
We should keep testing code inside the test project and not include it in CSP itself.

Reply:
I was more thinking of this as a way even clients could specifically trigger a test message rather than just our own testing since they would otherwise be waiting for some other log message to be able to verify. If we remove the no export on the `LogMsg` function then this would become unnecessary anyways.

Reply:
I think that using some other function that we know sends a log message would be a better idea here, as `LogTestMessage` is unlike any other code we have in CSP and is only ever probably going to be used by us. Having a public function for testing functionality feels wrong.

Reply:
It feels like it would be an ugly way of testing the log system to have a dependency on calling something else that we know, for now, logs a message. We couldn't be sure that in the future it would still do so.

Reply:
Would it help if I renamed this to something like `LogDefaultMessage`?

Reply:
Since the actual logging functionality is outside the scope of this ticket I'm going to just remove this and the test that I introduced.